account-kit/signer/src/session/manager.ts

```diff
@@ -7,7 +7,12 @@ import {
 } from "zustand/middleware";
 import { createStore, type Mutate, type StoreApi } from "zustand/vanilla";
 import type { BaseSignerClient } from "../client/base";
-import type { User } from "../client/types";
+import type {
+  AlchemySignerClientEvent,
+  AlchemySignerClientEvents,
+  AuthenticatingEventMetadata,
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'AuthenticatingEventMetadata' is defined but never used.
<!-- __reviewdog__:ChBiNmZkZjcyYjllMjQ0MWY2EgZyZGpzb24= -->